beam/www/beam/pages/Repack.vue

```diff
@@ -8,24 +8,293 @@
 		</template>
 	</Navbar>
 
-	<!-- <scale>
-		<readout :mqtt="" />
-	</scale> -->
+	<div class="repack">
+		<div class="container">
+			<template v-if="itemList">
+				<div class="dd-container">
+					<ADropdown
+						label="Item to Repack"
+						:items="itemList"
+						v-model="currentItem.item_code"
+						:isAsync="true"
+						:filterFunction="loadItems" />
+					<BeamBtn class="clear-button" @click="clearCurrentItem('item_code')"> X </BeamBtn>
+				</div>
+				<div class="dd-container">
+					<BeamBtn class="clear-button" @click="substractCurrentItem"> - </BeamBtn>
+					<ANumericInput label="Quantity" v-model="currentItem.qty" />
+					<BeamBtn class="clear-button" @click="addCurrentItem"> + </BeamBtn>
+				</div>
+				<div class="dd-container">
+					<ADropdown label="BOM (Optional)" :items="bomList" v-model="currentItem.bom" />
+					<BeamBtn class="clear-button" @click="clearCurrentItem('bom')"> X </BeamBtn>
+				</div>
+			</template>
+			<div class="dd-container">
+				<ADropdown label="Source Warehouse" :items="warehouseList" v-model="stockEntry.from_warehouse" />
```

Comment:
Should the selection map to the parent document, or to the individual items? This might only be a concern for repacks with multiple input/output items that may exist in different warehouses.

Reply:
Parent document. We can only get so sophisticated in a mobile view.

Reply:
@agritheory @lauty95 Then I think we should validate/confirm when a different warehouse is detected, either through scan or selection maybe?